packages/apputils-extension/src/themesplugins.ts

```diff
@@ -165,60 +220,40 @@ export const themesPaletteMenuPlugin: JupyterFrontEndPlugin<void> = {
             args: { isPalette, theme }
           });
         });
+
+        // sep
         themeMenu.addItem({ type: 'separator' });
 
         // toggle scrollbar theming
         themeMenu.addItem({ command: CommandIDs.themeScrollbars });
-        themeMenu.addItem({ type: 'separator' });
 
-        // increase/decrease code font size
-        themeMenu.addItem({
-          command: CommandIDs.incrFontSize,
-          args: {
-            label: trans.__('Increase Code Font Size'),
-            key: 'code-font-size'
-          }
-        });
-        themeMenu.addItem({
-          command: CommandIDs.decrFontSize,
-          args: {
-            label: trans.__('Decrease Code Font Size'),
-            key: 'code-font-size'
-          }
-        });
+        // sep
         themeMenu.addItem({ type: 'separator' });
 
-        // increase/decrease content font size
+        // make modifications to the various fonts
         themeMenu.addItem({
-          command: CommandIDs.incrFontSize,
-          args: {
-            label: trans.__('Increase Content Font Size'),
-            key: 'content-font-size1'
-          }
+          type: 'submenu',
+          submenu: fontMenu(
+            trans.__('Code Font'),
+            'code-font-family',
+            'code-font-size'
+          )
         });
         themeMenu.addItem({
-          command: CommandIDs.decrFontSize,
-          args: {
-            label: trans.__('Decrease Content Font Size'),
-            key: 'content-font-size1'
-          }
+          type: 'submenu',
+          submenu: fontMenu(
+            trans.__('Content Font'),
+            'content-font-family',
+            'content-font-size1'
+          )
         });
-        themeMenu.addItem({ type: 'separator' });
-
-        // increase/decrease ui font size
         themeMenu.addItem({
-          command: CommandIDs.incrFontSize,
-          args: {
-            label: trans.__('Increase UI Font Size'),
-            key: 'ui-font-size1'
-          }
-        });
-        themeMenu.addItem({
-          command: CommandIDs.decrFontSize,
-          args: {
-            label: trans.__('Decrease UI Font Size'),
-            key: 'ui-font-size1'
-          }
+          type: 'submenu',
+          submenu: fontMenu(
+            trans.__('UI Font'),
```

Comment:
Not sure about the use of UI, is this consistently use on other places?

Maybe `Interface Font` is another option that does not rely on shorthands?

Reply:
Though I agree that the term UI might be opaque for some, `Interface Font` is too generic and vague. For now, let's just keep things simple and continue to refer to this font the same way we do throughout the codebase and the css, as `UI font`